frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -73,6 +73,7 @@
 
         vm.isChallengeLeaderboardPrivate = false;
         vm.previousPublicSubmissionId = null;
+        vm.participantTeamDetail = "";
```

Comment:
Not a big fan of defining empty string. Maybe just declare it and don't set any value.